runtime/basilisk/src/lib.rs

```diff
@@ -351,7 +351,7 @@ parameter_types! {
 
 impl pallet_transaction_multi_payment::Config for Runtime {
 	type Event = Event;
-	type AcceptedCurrencyOrigin = MajorityTechCommitteeOrRoot;
+	type AcceptedCurrencyOrigin = EnsureRoot<AccountId>;
```

Comment:
i suspect this change is not related to the PR

Reply:
Oh. Missed that. I was explaining origin concept to audit guys. Forgot to revert.

Reply:
reverted